md/release-notes.md

```diff
@@ -1,18 +1,143 @@
+
+### Stability of files throughout revisions
+Updating timers, business data, data dependencies, messages, pools, contract inputs, parameters, expressions, connectors, and documents, used to create additional UUID (Universal Unique Identifier) that resulted in false positives when comparing two revisions of the same file (using Git or SVN). This has been improved to make only real changes stick out when using a Diff Tool.
+
+<a id="rest-timeout"/>
+
+### REST connector Read timeout is configurable
+In the REST connector, it is now possible to set the connection timeout (during connection) and the socket timeout (lack of answer of the distant machine after connection).
+The default value is one minute for both, the values that were hard-coded before.
+
+<a id="ldap-error"/>
+
+### LDAP synchronizer manages errors and continues on fail
+Better error management during LDAP synchronization.
+
+<a id="clause-in"/>
+
+### BDM query parameters accept arrays
+It is now possible to query for multiple values of an attribute in the BDM objects.
+
+<a id="translate-expression"/>
+
+### In the UI Designer, strings in JavaScript expressions can now be translated
+When writing the content of a variable of type JavaScript Expression, autocompletion now offers a new service for translation; uiTranslate().
+
+<a id="metadata"/>
+
+### Add a display name and display description to a page, form, or layout from the Editor
+Display name and description are used when the page, form, or layout are imported in the portal's Resources page.
+They could be edited by opening the page.properties file. Now, they can be directly edited in the page, form, or layout editor of the UI Designer, by clicking on the new "Tag" button.
+
+<a id="bonita-calls"/>
+
+### Get a summary of the calls to Bonita REST API made in a page, form, or layout
+Also by clicking on the "Tag" button, you can get a summary of all REST API calls made by the current artifact to the Bonita REST API.
+This helps validating if all expected calls are well implemented in the artifact in its current state.
+
+<a id="bonita-theme"/>
+
+### New Living Applications theme
+A new theme is available for any Living Application you create: "bonita-default-theme". 
+
+<a id="operations-apiaccessor"/>
+
+### Warning on best practices when using API in operations
+In diagram operations, the right operand should only use the apiAccessor with read-only methods. After helping several customers with scripts calling the apiAccessor with "write"-type methods, we have created a warning in the studio, when detecting the apiAccessor is called in an opreation script.
 
 <a id="technology-updates"/>
 
 ## Technology updates
 
+<a id="kerberos"/>
+
+### Integration of SSO Kerberos (Teamwork, Efficiency, Performance edition)
+Bonita works with Single-Sign-On (SSO) solutions using Kerberos protocol:
+- Use standard authentication service
+- Connect to Bonita Living Applications with company credentials (per tenant configuration)
```

Comment:
Why just "Bonita Living application" ? Shouldn't we say "Bonita Portal and Living Applications" ?

I believe "- Per tenant configuration" should be on a dedicated line

Reply:
I agree for the portal. Changed. 
OK for the per tenant config. Changed.
I've copied and pasted what was written for SAML. I'll change it there too then.